### ISSUE TITLE
Expired / unavailable project domain

### ISSUE THREAD

**Issue by h1z1** [mozilla/serviceworker-cookbook#333]
The domain https://serviceworke.rs/   is either invalid and no one noticed, or expired / hijacked.   Either way, it doesn't resolve.

Not sure where to report this tbh.  Technically it's a security issue as well assuming mozilla doesn't own the domain?

Was also reported https://github.com/mdn/content/issues/10666

**Comment by dipunj** [comment]
in the meanwhile can we have this site hosted at serviceworkers.github.io or serviceworkers.mozilla.org?

**Comment by marco-c** [comment]
@dipunj yes we should, but unfortunately there is no one currently maintaining this project. There is a PR to move the domain, but the build is currently failing so even if we merge it the fix won't get deployed.

**Comment by h1z1** [comment]
Appears this has been fixed but absolutely zero references to any issues nor even commit message. 

https://github.com/mdn/content/commit/e99085ad8239149e3df3758be296d0b5b4a4ba47


To be _very_ clear, I have nothing to do with this domain nor do I know anything about the registration.

I can't help but wonder (and laugh) what the long term stability of an organization which brings in millions but is unable to update a simple URL due to their own bureaucracies.  

The domain is now being squatted by someone (or something), else entirely.

```
domain name: serviceworke.rs
Domain status: Inactive https://www.rnids.rs/en/domain-name-status-codes#Inactive
Domain status: serverHold https://www.rnids.rs/en/domain-name-status-codes#ServerHold
Domain status: pendingUpdate https://www.rnids.rs/en/domain-name-status-codes#PendingUpdate
Registration date: 17.12.2021 01:00:06
Modification date: **06.01.2022 01:00:26**
Expiration date: 17.12.2022 01:00:06
Confirmed: no
Registrar: Stanco d.o.o.


Registrant: Individual

Administrative contact: Individual

Technical contact: Individual


DNS: ns1.istanco.com - 
DNS: ns2.istanco.com - 


DNSSEC signed: no
```

cheers

**Comment by marco-c** [comment]
@h1z1 this is an open source project with no maintainer, feel free to contribute a fix to the build and I will happily deploy it. The domain is lost, it has been registered by someone else. Given that the project is unmaintained and so likely doesn't use the latest and greatest SW features, it's nothing to worry about.

**Comment by h1z1** [comment]
Mozilla is not just an opensource project.  If you actually followed the link in my reply you'd have seen it was corrected already.

**Comment by marco-c** [comment]
@h1z1 I'm talking about *this* project. The MDN link was corrected to point to this repository, but this repository is still deployed in a way that forcibly redirects to serviceworke.rs, which is no longer owned by us.
#334 would fix that, but the build is failing and there is no one maintaining this project to fix that.
